java/src/Ice/src/main/java/com/zeroc/Ice/Exception.java

```diff
@@ -12,10 +12,18 @@
 public abstract class Exception extends RuntimeException implements Cloneable {
   public Exception() {}
 
+  public Exception(String message) {
+    super(message);
+  }
+
   public Exception(Throwable cause) {
```

Comment:
No, I deleted it.

I thought that `UserException` derived from this class, but apparently it doesn't in Java (because of checked vs unchecked exceptions).